pkg/service/aerospoike_service.go

```diff
@@ -0,0 +1,33 @@
+package service
+
+import (
+	"fmt"
+	"strings"
+
+	as "github.com/aerospike/aerospike-client-go/v7"
+	"github.com/aerospike/backup/pkg/model"
+)
+
+const namespaceInfo = "namespaces"
+
+func getAllNamespacesOfCluster(cluster *model.AerospikeCluster) ([]string, error) {
+	policy := as.NewClientPolicy()
```

Comment:
It's better to have a `ToClientPolicy` method for `AerospikeCluster`, where it will utilize all of its fields.